src/route/RoutePath.jsx

```diff
@@ -24,6 +25,7 @@ export default function RoutePath() {
       <Route path="/Logout" element={<Logout />} />
       <Route path="/Favourites" element={<Favourites />} />
       <Route path="/WatchLater" element={<WatchLater />} />
```

Comment:
Favourite and watch later should be private route

Reply:
They are going to be implemented after the p0 screens are implemented